doc/sphinx-guides/source/installation/config.rst

```diff
@@ -866,9 +866,9 @@ For example:
 
 ``cp <your key file> /usr/local/payara5/glassfish/domains/domain1/files/googlecloudkey.json``
 
-.. _Archiving API Call:
+.. _Archiving API Calls:
 
-API Call
+API Calls
 ++++++++
```

Comment:
Hmm, I thought I was doing enough by checking to see if github could properly render things (which it did with these), so that's not enough. Is there a place where I can check for a given PR to see the errors you're finding, or do I need to get set up to build the guides locally? (is it an ansible option?)